frontend/packages/process-editor/src/utils/processEditorUtils/processEditorUtils.test.ts

```diff
@@ -12,4 +14,44 @@ describe('processEditorUtils', () => {
       expect(result).toBeFalsy();
     });
   });
+
+  describe('updateDataTaskTrackingLists', () => {
+    it('calls the methods to add item to primary list and remove from secondary list', () => {
+      const updatePrimaryList = jest.fn();
+      const updateSecondaryList = jest.fn();
+      const itemToAdd: BpmnDetails = {
+        id: '1',
+        name: 'test',
+        taskType: 'data',
+        type: BpmnTypeEnum.Task,
+      };
+      const secondaryList: BpmnDetails[] = [
+        { id: '1', name: 'test', taskType: 'data', type: BpmnTypeEnum.Task },
```

Comment:
If the idea of this test is to simulate that one removes a newly added data task, maybe it is clearer (and closer to a realistic scenario) if we call the `itemToAdd` `itemToRemove` instead and the secondary list for `dataTasksAdded`? And also initialize the list with the same const `itemToRemove` 🤔